src/app/home.component.html

```diff
@@ -87,6 +87,7 @@
   </stache-page-anchor>
 
   <stache-image
+    alt="This is an image"
```

Comment:
Screen readers might have issue with this (`alt` isn't just for images). http://webaim.org/techniques/alttext/#conclusion

I'd prefer `imageAlt` or `alternateText`, instead.